test/integration/local/storage.test.js

```diff
@@ -0,0 +1,36 @@
+const { queryEntities } = require('../../../ffc-pay-event/storage')
+jest.mock('@azure/data-tables', () => {
```

Comment:
I'd maybe move this to unit test as it's not really a local integration test, ie it's not using containerised local dependencies, it's mocking.

Given functions are pretty much exclusively going to be unit tests, then until contract tests are added, it would probably be simpler to just have `test` folder without subfolders?